PublishParasoftTestResults/publishtestresults.ts

```diff
@@ -478,7 +494,7 @@ function isValidPort(port : any) {
 
 async function handleRuleDocsPromises(promises: Promise<any>[]) {
     await Promise.all(promises).then((response) => {
-        if (ruleDocUrlMap.size !== ruleAnalyzerMap.size) {
+        if (response.length > 0 && ruleDocUrlMap.size !== ruleAnalyzerMap.size) {
```

Comment:
When promises is empty, avoid repeating warnings. Alternatively, before calling the handleRuleDocsPromises() method, verify whether the DTP service is started